doc/2/api/controllers/server/capabilities/index.md

```diff
@@ -4,20 +4,21 @@ type: page
 title: capabilities
 ---
 
-# Capabilities
+# capabilities
 
 
 
 Returns the current Kuzzle capabilities.
 
-This route should only be accessible to administrators, as it might return sensitive information about the backend.
+This action is accessible by all users, the purpose of this action is to inform the client what kind of limits or behavior they should expect from Kuzzle.
 
 ---
 
 ## Query Syntax
 
 ### HTTP
 
+<SinceBadge version="2.16.6"/>
```

Comment:
You need to use the `auto-version` placeholder since you cannot predict the version number of the release your feature will be on. This placeholder is replaced by the release tool in the release process.